types/karma-sinon-chai/test/karma-sinon-chai-global-tests.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference types="chai" />
 /// <reference types="sinon-chai" />
+import "chai/register-should";
+import * as sinon from 'sinon';
```

Comment:
I had to add this to greenify the build, but I don't think it really belongs to this change (or is even correct...? If I understand correctly, `sinon` types should be available ambiently, which is what this test is checking?)

Reply:
By adding an import here, you're turning the file into a module when it was previously a global script. You should use a `reference` directive to `chai/register-should` here and in other places that are not modules. (and that would have to be the recommendation for other global users).